src/content/en/fundamentals/performance/rail.md

```diff
@@ -132,21 +142,16 @@ To evaluate your site against RAIL metrics, use the Chrome DevTools [Timeline to
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms.</td>
-      <td data-th="User Test">User taps on an icon or button (for example, opening the nav menu, tapping Compose).</td>
+      <td data-th="User Test">User taps a button (for example, opening navigation).</td>
     </tr>
     <tr>
       <td data-th="RAIL Step"><strong>Response</strong></td>
-      <td data-th="Key Metric">Input latency (from tap to paint) < 16ms.</td>
-      <td data-th="User Test">User drags their finger and the app's response is bound to the finger position (for example, pull to refresh, swiping a carousel).</td>
-    </tr>
-    <tr>
-      <td data-th="RAIL Step"><strong>Animation</strong></td>
       <td data-th="Key Metric">Input latency (from tap to paint) < 100ms for initial response.</td>
       <td data-th="User Test">User initiates page scroll or animation begins.</td>
     </tr>
```

Comment:
This is a bit hairy actually. Based on the folks I asked to try out a test page, most people actually find about 50ms the threshold for when touch drag or scroll start becomes too slow, which doesn't fit nicely into the RAIL model.

I'd be happy just removing the scroll start / drag start case here.